### PR TITLE
Fixes up and started

### PR DESCRIPTION
**Pull request by pdufour**

### CODE REVIEW

**Comment by pdufour** [comment]
Reposting comments since I recreated repo as a private repo.

@monokrome 

> @sjolicoeur I think this needs to be reverted as it still needs reviewed. @pdufour and I spoke about it briefly and we feel like some things should be squashed / fixed before it gets merged.
> 
> Will anything break on master if it's reverted?

@pdufour

> Yeah I think it should be required to have a reviewer on a change to a core part of our system (payments), even if the changes seem minor.

**Comment by monokrome** [comment]
Why a private repo?...

**Comment by pdufour** [comment]
It was discussed in Slack. Their was information about our DB schema that
was being made public.

**Comment by monokrome** [comment]
Yeah, but I mentioned that you can fix it with a branch filter

**Comment by monokrome** [comment]
@sjolicoeur Did the squash require a force push so that it removed our schema data from history?

**Comment by sjolicoeur** [comment]
I had to spin up a new project to redo the migration and yes force push

**Comment by monokrome** [comment]
@pdufour Any objection to making this public again?

**Comment by pdufour** [comment]
sounds okay @monokrome

**Comment by sjolicoeur** [comment]
@pdufour  is this good to merge?

**Comment by pdufour** [comment]
I'm not really sure what it's doing @sjolicoeur. @monokrome did you review?

File: amazon_payments/views.py [hunk 1]
```diff
@@ -540,6 +543,12 @@ def post(self, request, *args, **kwargs):
             order_total = self.get_order_totals(
                 self.request.basket,
                 shipping_method=shipping_method)
+
+            ### by sjf
```
Comment (monokrome) [inline amazon_payments/views.py]: @sjolicoeur Why?

**Comment by sjolicoeur** [comment]
@monokrome  removed

File: amazon_payments/views.py [hunk 2]
```diff
@@ -540,6 +543,12 @@ def post(self, request, *args, **kwargs):
...
+            request.basket.calculate_tax(
```
Comment (monokrome) [inline amazon_payments/views.py]: @sjolicoeur Why multi-line here?

File: amazon_payments/views.py [hunk 3]
```diff
@@ -540,6 +543,11 @@ def post(self, request, *args, **kwargs):
             order_total = self.get_order_totals(
                 self.request.basket,
                 shipping_method=shipping_method)
+
+            request.basket.calculate_tax(
+                shipping_address
+            )
+            ###
```
Comment (monokrome) [inline amazon_payments/views.py]: @sjolicoeur Why this comment? I feel like whitespace is better than `###` in every case I've seen it. Maybe I don't know what you're denoting with the `###` comments?

Comment (sjolicoeur) [inline amazon_payments/views.py]: visual break mostly I'll remove

Comment (monokrome) [inline amazon_payments/views.py]: @sjolicoeur Can you take a look at this as well? https://github.com/britco/django-oscar-amazon-payments/pull/1#discussion_r34180422

**Comment by monokrome** [comment]
@sjolicoeur :cake: :D

**Pull request merged**